Rawl_B502/mínugud_senchusa_laigin_and_so_sís.ttl

```diff
@@ -0,0 +1,1264 @@
+    rdfs:comment "Cuthraige .i. Síl Fergusa Cuthig & Cuirc Librén.";
+    rel:descendantOf <#CormacLosc>.
+
+<#CuircLibrén>
+    a foaf:Person;
+    irishRel:genName "Cuirc Librén";
+    irishRel:nomName "Corc Librén";
+    rdfs:comment "Cuthraige .i. Síl Fergusa Cuthig & Cuirc Librén.";
+    rel:descendantOf <#CormacLosc>.
+
+<#MessinCorb>
+    a foaf:Person;
+    irishRel:nomName "Messin Corb";
+    rel:childOf <#ConChorb>;
+    rel:childOf <#Ethne>;
+    owl:sameAs <http://example.com/LL/lagin.ttl#MessiCorb>;
+    rdfs:comment "Dál Messin Corb in prím-sluinniud tánaisse. Airich insin .i. tánaise ríg ind fhir-sin. Argat bruinte díre a n-enechclainne.".
+
+<#CoirpreCruaid>
+    a foaf:Person;
+    irishRel:nomName "Coirpre Cruaid";
+    rel:childOf <#ConChorb>;
+    rel:childOf <#Ethne>;
+    owl:sameAs <http://example.com/LL/lagin.ttl#Carpre-e9186b00>;
+    rdfs:comment "Dál Cairpri Loingsig Bic in cethramad prím-slonnud Lagen. La h-Aradu Cliach atát.".
+
+<#Ethne>
+    a foaf:Person;
+    foaf:gender "female";
+    rel:childOf <#Múisc>;
+    rel:childOf <#CoirpreMúisc>;
+    rdfs:comment "ingen Múisc uel magis Corpri Múisc ut alii dicunt";
+    owl:sameAs <http://example.com/LL/lagin.ttl#Eithni>.
+
+<#Múisc>
+    a foaf:Person;
+    irishRel:genName "Múisc";
+    irishRel:nomName "Músc".
+
+<#CoirpreMúisc>
+    a foaf:Person;
+    irishRel:genName "Coirpre Múisc";
+    irishRel:nomName "Coirpre Músc";
+    owl:sameAs <http://example.com/LL/corco_duibni.ttl#CairpriMusc>.
+
+<#NiaCorb>
+    a foaf:Person;
+    irishRel:nomName "Nia Corb";
+    rel:childOf <#ConCorb>;
+    owl:sameAs <#NiadCuirb>;
+    rdfs:comment "Is dona cethrib bráithrib-sin ro chachain in file: 'Nia Corb Corbmac [...]'".
+
+<#Corbmac>
+    a foaf:Person;
+    irishRel:nomName "Corbmac";
+    rel:childOf <#ConCorb>;
+    owl:sameAs <#CormacLosc>;
+    rdfs:comment "Is dona cethrib bráithrib-sin ro chachain in file: 'Nia Corb Corbmac [...]'".
+
+<#Cairpre>
+    a foaf:Person;
+    irishRel:nomName "Cairpre";
+    rel:childOf <#ConCorb>;
+    owl:sameAs <#CoirpreCruaid>;
+    rdfs:comment "Is dona cethrib bráithrib-sin ro chachain in file: 'Nia Corb Corbmac [...]'".
+
+<#MessinCorb-cdd91f50>
+    a foaf:Person;
+    irishRel:nomName "Messin Corb";
+    owl:sameAs <#MessinCorb>;
+    rel:childOf <#ConCorb>;
+    rdfs:comment "Is dona cethrib bráithrib-sin ro chachain in file: 'Nia Corb Corbmac [...]'".
+
+<#ConCorb>
+    owl:sameAs <#ConChorb>.
+
+<#Luccraid>
+    a foaf:Person;
+    irishRel:nomName "Luccraid";
+    rel:descendantOf <#Riadda>;
+    rdfs:comment "[author of] 'Derb dal Cuirb [...]'".
+
+<#Riadda>
+    a foaf:Person;
+    irishRel:genName "Riadda".
+
+<#Cuirb>
+    a foaf:Person;
+    irishRel:genName "Cuirb";
+    owl:sameAs <#ConCorb>;
+    rdfs:comment "Unde Luccraid moccu Riadda dixit: 'Derb dal Cuirb [...]'".
+
+<#MessinCorb-8e098540>
+    a foaf:Person;
+    irishRel:nomName "Messin Corb";
+    owl:sameAs <#MessinCorb>;
+    rel:relationship <#Baccae>;
```

Comment:
I've changed it to "foaf:knows". It was not clear to me who "Baccae" is or how the other individuals in the poem relate to him/her ("in one peace with Baccae"?). So foaf:knows seems safe for now.